en/docs/develop-components/develop-proxy/develop-a-git-api-proxy.md

```diff
@@ -0,0 +1,166 @@
+# Develop an API Proxy from Git Repository
```

Comment:
Let's decide on https://github.com/wso2/docs-choreo-dev/pull/1696/files#r1976915437 and update this title and markdown file name accordingly if we are ok to go with the suggestion.